notes.md

```diff
@@ -0,0 +1,24 @@
+# development
+## setup
+```
+python -m venv venv
+venv\Scripts\activate
+python -m pip install --upgrade pip
+pip install -r requirements.txt
+```
+## for admin purposes saving & upgrading
+
+```
+venv\Scripts\activate
+call pip freeze > requirements.txt
+powershell "(Get-Content requirements.txt) | ForEach-Object { $_ -replace '==', '>=' } | Set-Content requirements.txt"
+call pip install -r requirements.txt --upgrade
+call pip freeze > requirements.txt
+powershell "(Get-Content requirements.txt) | ForEach-Object { $_ -replace '>=', '==' } | Set-Content requirements.txt"
+```
+## build & publish
+building package and deploying to pypi, must have pypi account
+```
+python -m build
+twine upload dist/*
+```
```

Comment:
As an outside observer, I would disagree that it does not add value; this helps document the steps to take in terms of consistency in publishing releases via pypi, which helps in terms of maintenance in the future should the need arise. The info may be better served as a GitHub wiki page or other marked documentation, instead of within the source, but certainly having it documented is a good thing.